core/src/util/usUtils.cpp

```diff
@@ -388,46 +388,43 @@ std::string GetLastErrorStr()
 #endif
 }
 
-static MsgHandler handler = 0;
-
-MsgHandler installMsgHandler(MsgHandler h)
-{
-  MsgHandler old = handler;
-  handler = h;
-  return old;
-}
-
-void message_output(MsgType msgType, const char *buf)
+void TerminateForDebug(const std::exception_ptr ex)
```

Comment:
This function is now never used. Would it make sense to call it in case of a framework error and a proprietary framework property being set? Should we call it in cases where we only issue a DIAG_LOG call and the error condition is "critical enough" ?